modular_zubbers/code/modules/mob/living/carbon/human/species/xeno.dm

```diff
@@ -1,3 +1,43 @@
 /datum/species/xeno
 	mutant_organs = list()
 	heatmod = 1
+	inherent_traits = list(
+		TRAIT_ADVANCEDTOOLUSER,
+		TRAIT_CAN_STRIP,
+		TRAIT_LITERATE,
+		TRAIT_MUTANT_COLORS,
+		TRAIT_NIGHT_VISION
+	)
+	species_language_holder = /datum/language_holder/xeno_hybrid
+
+/datum/species/xeno/get_species_description()
+	return list(
+		"A 'Xenomorph Hybrid' is a loose designation for a wide variety of creatures with predominantly \
+		Xenomorph-like characteristics, rather than a specific species in its own right. As Xenomorphs themselves are \
+		parasitic hybridisations of many different 'host' species, the main distinguishing factors between a regular \
+		Xenomorph and a Hybrid are a more humanoid body shape and stature with alien features, inert or defective hive \
+		organs and a less murderous demeanour. These are not the only documented differences, though they are the most frequent.",
```

Comment:
```suggestion
		organs, and a less murderous demeanor. These are not the only documented differences, though they are the most frequent.",
```

Reply:
English (Traditional) to English (Simplified) code review goes hard. US-based server, though, so it makes sense

I will fix it when I get home, GitHub Mobile won't let me